docs/developers/integrating_discord.md

```diff
@@ -0,0 +1,47 @@
+---
+id: integrating_discord
+title: Integrating Discord
+sidebar_label: Integrating Discord
+---
+The LMS offers basic Discord integration, allowing you to cache messages from any user who has linked their Discord profile from a specific server. Additionally, you can modify user roles.
+
+### Setting the configuration
+To set up Discord integration, you will need to configure the settings for your school.
+
+Here are the steps:
+
+#### Step 1: Create a Discord bot
+You'll need to visit [Discord developer portal](https://discord.com/developers/applications/) and generate a new application there. Once you've created the application, enable the Presence Intent, Server Members Intent, and Message Content Intent under Privileged Gateway Intents.
```

Comment:
It could be better to skip "You will need to" and start instruction details.

Intent options mentioned require user to first switch from "general information" (where he will arrive after creating app) to "bot" settings.

```suggestion
Visit [Discord developer portal](https://discord.com/developers/applications/) and generate a new application there. Then go to "Bot" settings and enable the Presence Intent, Server Members Intent, and Message Content Intent under Privileged Gateway Intents.
```